GSA/digitalgov.gov
# Remove Edit Button

## Summary

This Pr will remove the edit button that lives on the site that allows users the ability to suggest edits via github 

This will also remove the other edit button at the end of the pages along with the `last updated` functionality. 

Will also update the footer link to `Submit issue on GitHub` and link to the issues page for users to make and issue.

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-remove-edit-btn/)



### Solution

- Removed template page for the edit button 
- Removed both styles and js files that were associated with this button 
- Removed the forward in the global styles sheet 

### How To Test

1. Visit [preview link](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-remove-edit-btn/) and ensure that button is no longer present 
2. Also visit [other preview link]() can scroll to the bottom and check that edit and last updated are no longer present. 
3. Check that footer link now reads `Submit issue on GitHub` and link takes user to issues page. 
4.  Check that any mention of the edit button is removed from the repo 

---

### Dev Checklist



tagging supervisor for reference - @rayestrada
